services/src/main/java/org/apache/druid/cli/CliPeon.java

```diff
@@ -308,6 +308,9 @@ public Supplier<Map<String, Object>> heartbeatDimensions(Task task)
             builder.put(DruidMetrics.DATASOURCE, task.getDataSource());
             builder.put(DruidMetrics.TASK_TYPE, task.getType());
             builder.put(DruidMetrics.GROUP_ID, task.getGroupId());
+            if (task.getStatus() != null) {
+              builder.put(DruidMetrics.TASK_STATUS, task.getStatus());
```

Comment:
I don't think we should use this dimension as it corresponds to the final status of a task which only takes values `SUCCESS` or `FAILURE`. We could use `DruidMetrics.STATUS` or some other dimension instead.

Reply:
Fixed. Let me know if ok now.